### PR TITLE
Board room dev ratan

### PR DESCRIPTION
**Pull request by ratankhalsa92**

### CODE REVIEW

File: src/MediaPane/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  return (
+
+    <div className="flex stickyNotes" style={{position: 'absolute', width: '360px', bottom: '20px'}}>
+      {data.map(s => {
+        return <Radio value={s.value} onChange={onChange} name={'Data'} content={s.content} isSelected={selected} />;
+      })}
+
+      <div style={{ position: 'absolute', bottom: '10px', right: '20px'}}>
+        <CloseIcon />
```
Comment (ShreyasArunkumar) [inline src/MediaPane/index.tsx:48]: close Icon should take a `onClick`

File: src/Pin/index.scss [hunk 2]
```diff
@@ -1,54 +1,33 @@
...
-    & .board_ui__pin_red {
-      background-color: red;
-      color: white;
-    }
-
-    & .board_ui__pin_blue {
-      background-color: blue;
-      color: black;
-    }
+body {
```
Comment (ShreyasArunkumar) [inline src/Pin/index.scss:1]: never target global classes

File: src/Pin/index.scss [hunk 3]
```diff
@@ -1,54 +1,33 @@
...
-    }
+.pin_ui {
+  width: 900px;
+  margin: 0 auto;
+  color: #000;
+  background-color: #fff;
+  font-family: "Montserrat", sans-serif;
+  padding-bottom: 10px;
+  border: 1px dashed #c8c8c8;
+  margin-bottom: 20px;
```
Comment (ShreyasArunkumar) [inline src/Pin/index.scss:13]: y not use this on the line 7 as `margin: 0 auto 20px`

File: src/Pin/index.tsx [hunk 4]
```diff
@@ -1,22 +1,31 @@
-import React from 'react';
-import { IPin } from '../Models';
-
+import React, { useState } from 'react';
+import './index.scss';
```
Comment (ShreyasArunkumar) [inline src/Pin/index.tsx:2]: same here

File: src/PinArchive/index.scss [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      position: absolute;
+      top: 2px;
+      right: 7px;
+      cursor: pointer;
+    }
+
+    & textarea {
+      padding: 10px;
+      border: none;
+      width: 50px;
```
Comment (ShreyasArunkumar) [inline src/PinArchive/index.scss:17]: correct me if I'm wrong. Don't u think this is very small

File: src/PinArchive/index.scss [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+      letter-spacing: 1.2px;
+      margin: 5px;
+      box-shadow: 0 0 3px 0 rgba(0, 0, 0, 0.25);
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    & .board_ui__pin_yellow {
```
Comment (ShreyasArunkumar) [inline src/PinArchive/index.scss:29]: all the below class can be targeted as 

````scss
&__yellow {

}

&__red {
}

...
````

File: src/PinArchive/index.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+export function Pin({ pinProps, onChange, onDelete }: Props) {
+  function pinContentModified(body: string) {
+    onChange({...pinProps, body});
+  }
+
+  return (
+    <div className='board_ui__pin'>
+      <textarea className={`board_ui__pin_${pinProps.color ? pinProps.color : 'yellow'}`} onChange={(e: any) => pinContentModified(e.target.value)} value={pinProps.body}></textarea>
```
Comment (ShreyasArunkumar) [inline src/PinArchive/index.tsx:18]: this triggers a on change on every character u type!
Is this how it is requested!

File: src/PinEditor/index.scss [hunk 8]
```diff
@@ -0,0 +1,69 @@
+.overlay {
```
Comment (ShreyasArunkumar) [inline src/PinEditor/index.scss:1]: try renaming to the board_ui__background_overlay

File: src/PinEditor/index.scss [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+      transition: background-size 0.25s ease-in;
+      background-repeat: no-repeat;
+
+      &:focus {
+        outline: none;
+      }
+    }
+  }
+
+  &__title .close_btn_wrapper {
```
Comment (ShreyasArunkumar) [inline src/PinEditor/index.scss:44]: y not define inside the title

File: src/PinEditor/index.tsx [hunk 10]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './index.scss';
```
Comment (ShreyasArunkumar) [inline src/PinEditor/index.tsx:2]: same here

File: src/_CreatePinEdtitor-tobedeleted/index.tsx [hunk 11]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import MediaPane from '../MediaPane';
+import './index.scss';
```
Comment (ShreyasArunkumar) [inline src/_CreatePinEdtitor-tobedeleted/index.tsx:3]: same here

File: src/_CreatePinEdtitor-tobedeleted/index.tsx [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+import '../../src/scss/_components.scss';
+
+export default function CreatePinEditor() {
+
+  return (
+    <div className="createPinEditor_ui">
```
Comment (ShreyasArunkumar) [inline src/_CreatePinEdtitor-tobedeleted/index.tsx:9]: We are not using camel case for naming classes

File: stories/components/createPinEditor.stories.tsx [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+];
+
+function PinController() {
+  const [pinData, setPinData] = useState(initialData);
+
+  const [currentSelected, setCurrentSelected] = useState({id: 2, title: "Team meeting over", description: "Some dummy text."});
+  const [showPin, setShowPin] = useState(false);
+
+  function onSubmit(data: any){
+    console.log(data);
```
Comment (ShreyasArunkumar) [inline stories/components/createPinEditor.stories.tsx:19]: remove this below updates the state

File: src/PinEditor/index.tsx [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+import CloseIcon from '../SvgIcons/CloseIcon';
+
```
Comment (amitrajput1992) [inline src/PinEditor/index.tsx:4]: reduce whitespace

File: src/PinEditor/index.tsx [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+interface Props {
+  data: Data,
+  onSubmit: (data: Data) => void,
+  closeEditor: () => void
+}
+
+function PinEditor({data, onSubmit, closeEditor}: Props) {
+
```
[the remark this answers is not in the document]
Comment (amitrajput1992) [reply src/PinEditor/index.tsx:19]: onChangeBody:
construct a new object of type:
const update = {
id: ,
title: state.title,
body: state.body
};

props.onChange(update);

File: src/PinEditor/index.scss [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  // opacity: 0.75;
+  background-color: #ddd;
+}
+
+.pin_editor {
```
Comment (amitrajput1992) [inline src/PinEditor/index.scss:11]: add a parent class `.board-ui`

**Pull request merged**